### DISCUSSION TITLE
2024/10/09/post-reaction

### DISCUSSION THREAD

**Discussion by giscus** [PENG-AO/PENG-AO.github.io#23] [Comment]
# 2024/10/09/post-reaction

2024.11.03：本站已将评论系统由utterances迁移至giscus。giscus基于GitHub Discussion构建，支持对于整体的reaction。因此本文内容已不在本站使用，仅供读者参考。

https://peng-ao.github.io/2024/10/09/post-reaction.html

**Comment by ppaul12** [comment]
[Giscus](https://giscus.app)评论测试。

**Reply by ppaul12** [reply]
回复测试。
